### ISSUE TITLE
Mac OS Sierra 10.12 : Session not lunching after getting patched 

### ISSUE THREAD

**Issue by hugoe8** [Floris497/mac-pixel-clock-patch-V2#75]
Hi! 
First : Excuse me for my english, i'm in fact french but i'll try to make my best for you

So, i was using this patch on El capitan for my 21:9 screen and it was working well ! All good
Since i updated to Mac os Sierra, i wanted to patch again so that i can have my screen with the 2560x1440 but after Disabling SIP, Patching with Core display and rebooting ... when i try to open my session ( or account , i don't know the word in english ) , the loading freeze and i can't open my session, the only way is to go in recovery mode and reinstall Mac OS. 
I'am blocked here, my screen is quite ugly because displaying 1920x1080 on 21:9 
I know that the patch work on sierra with some people but i don't understand what they are saying ( I think that i read that i must change something in the patch to get i work ) 

Please Help me because your patch was so good on El Capitan, and i don't know another way to fix my screen

Thanks

**Comment by hugoe8** [comment]
Also, i have a MacBook Air 2015 and the Screen is an LG 25UM57 ; To link them i use a Thunderbolt to HDMI and then an HDMI cable to the screen

**Comment by Floris497** [comment]
Ehm.. yes, when you log in you get a screen setting.. sometimes different from the one used om the login screen.. try to change it with remote desktop. Only the screen should go blank, your mac should still work. You can also set a keyboard shortcut in switchxres that can for instance change the resolution to 1080p a resolutions from where you can try to set the native resolution of your monitor. Hope this helps. And your english is fine :)

**Comment by hugoe8** [comment]
Hi Floris,
Thank for the reply
I think that you didn't understood the issue when lunching, 
https://cloud.githubusercontent.com/assets/22396844/18790360/ccfb80a8-81ae-11e6-981c-54cfe71b2927.png
This is not my screenshot but it is an exemple : when i am on this screen, i put my password and then it start to load but like 10 sec after the load stop, the load bar is half filled but it don't go more

**Comment by Floris497** [comment]
So before you get to the login screen?

**Comment by hugoe8** [comment]
No, i succeed in joining the login screen, i enter the password and then it start to load and it freeze on the loading screen

**Comment by Floris497** [comment]
Yep. So mostly the screen freezes and not the computer.. if you acces it with remote desktop it mostly will work, from there you can set a working resolution. Are you sire your computer doesn't work at all after this screen?

**Comment by hugoe8** [comment]
I don't know because it stay on this loading page and the only thing that i can do is shutting it down by pressing the power button ... You think that it is only a resolution problem and if i go with remote desktop in the setting to change the resolution during the freeze it could help ?

**Comment by Floris497** [comment]
If the loading thing is still moving then it's a different problem. If it stops moving it's a resolution problem

**Comment by hugoe8** [comment]
The loading thing still moving but it stops at one point

**Comment by Floris497** [comment]
When it stops display settings are altered.. Did you try using any Remote Desktop software from a different PC or from your phone?
